i18n/locales/cn.json

```
@@ -99,7 +99,8 @@
   },
   "searchResultsDetails": {
     "speaks": "这位医师可以说",
-    "contact": "联系"
+    "contact": "联系",
+    "lastUpdate": "Last Update"
```

Let's make this past tense

```suggestion
    "lastUpdate": "Last Updated"
```